templates/gallery/image/edit.html

```diff
@@ -47,13 +47,13 @@
             {% trans "Code markdown pour insérer cette image" %} : <br>
 
             {% trans "Taille normale" %} :
-            <input type="text" value="![{{ image.legend }}]({{app.site.url}}{{ image.physical.url }})" readonly onclick="this.select()"> <br>
+            <input type="text" value="![{{ image.legend }}]({{ image.physical.url }})" readonly onclick="this.select()"> <br>
```

Comment:
C'est quoi `onclick="this.select()"` ?

`{{ image.legend }}` donne `None` si on n'a pas renseigné de légende, ce qui n'est pas ce qu'on attend.